app/src/main/java/com/github/se/orator/ui/profile/ProfileScreen.kt

```diff
@@ -124,147 +135,193 @@ fun ProfileScreen(navigationActions: NavigationActions, profileViewModel: UserPr
       backgroundColor = MaterialTheme.colorScheme.background) { innerPadding ->
         Column(
             modifier =
-                Modifier.fillMaxSize().padding(innerPadding).padding(AppDimensions.paddingMedium),
-            horizontalAlignment = Alignment.CenterHorizontally) {
-              userProfile?.let { profile ->
-                Box(
-                    modifier =
-                        Modifier.fillMaxWidth()
-                            .height(AppDimensions.profileBoxHeight)
-                            .padding(top = AppDimensions.paddingXXXLarge),
-                    contentAlignment = Alignment.TopCenter) {
-                      // Background "card" behind the profile picture
-                      Card(
-                          modifier =
-                              Modifier.fillMaxWidth(0.95f).height(AppDimensions.profileCardHeight),
-                          backgroundColor = MaterialTheme.colorScheme.surfaceVariant,
-                          elevation = AppDimensions.elevationSmall) {}
+                Modifier.fillMaxSize()
+                    .padding(innerPadding)
+                    .padding(AppDimensions.paddingMedium)
+                    .verticalScroll(rememberScrollState()), // Enable scrolling
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.spacedBy(AppDimensions.paddingMedium),
+        ) {
+          userProfile?.let { profile ->
+            /**
+             * Displays user profile information, including profile picture, name, streak, and bio.
+             */
+            Box(
+                modifier =
+                    Modifier.fillMaxWidth()
+                        .height(AppDimensions.profileBoxHeight)
+                        .padding(top = AppDimensions.paddingXXXLarge),
+                contentAlignment = Alignment.TopCenter) {
+                  // Background "card" behind the profile picture
+                  Card(
+                      modifier =
+                          Modifier.fillMaxWidth(0.95f)
+                              .height(AppDimensions.profileCardHeight)
+                              .shadow(
+                                  elevation = AppDimensions.elevationSmall,
+                                  shape = RoundedCornerShape(size = AppDimensions.statusBarPadding),
+                                  clip = false)
+                              .background(
+                                  colors.surfaceVariant,
```

Comment:
The color of the card is not taken into account : 

Version on the main : 

<p align="center"><img src="https://github.com/user-attachments/assets/e6011347-4c61-4c80-9ea8-976ac0929434" height="500"></p>

This version : 
<p align="center"><img src="https://github.com/user-attachments/assets/2501ce7d-188d-4005-a921-c875a47cab1d" height="500"></p>